Schema/V1/no.ks.fiks.arkiv.v1.arkivering.arkivmelding.oppdater.xsd

```diff
@@ -29,19 +29,23 @@
     <xs:complexType name="dokumentbeskrivelseOppdatering">
         <xs:sequence>
             <xs:element name="systemID" type="n5mdk:systemID" />
+            <xs:element name="dokumenttype" type="n5mdk:dokumenttype" minOccurs="0"/>
             <xs:element name="dokumentstatus" type="n5mdk:dokumentstatus" minOccurs="0"/>
             <xs:element name="tittel" type="n5mdk:tittel" minOccurs="0"/>
             <xs:element name="beskrivelse" type="beskrivelseOppdateringer" minOccurs="0"/>
             <xs:element name="forfatterOppdateringer" type="forfatterOppdateringer" minOccurs="0" maxOccurs="unbounded"/>
             <xs:element name="opprettetDato" type="n5mdk:opprettetDato" minOccurs="0"/>
             <xs:element name="opprettetAv" type="n5mdk:opprettetAv" minOccurs="0"/>
+            <xs:element name="dokumentmedium" type="n5mdk:dokumentmedium" minOccurs="0"/>
             <xs:element name="tilknyttetRegistreringSom" type="n5mdk:tilknyttetRegistreringSom"/>
             <xs:element name="dokumentnummer" type="n5mdk:dokumentnummer" minOccurs="0"/>
             <xs:element name="tilknyttetDato" type="n5mdk:tilknyttetDato" minOccurs="0"/>
             <xs:element name="tilknyttetAv" type="n5mdk:tilknyttetAv" minOccurs="0"/>
             <xs:element name="partOppdateringer" type="partOppdateringer" minOccurs="0"/>
+            <xs:element name="merknad" type="arkivmelding:merknad" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
This one must be as for merknad in mappe and registrering (<xs:element name="merknadOppdateringer" type="merknadOppdateringer" minOccurs="0"/>)

Reply:
Ja, jeg mistenkte det 👍